simulation/traffic_simulator/include/traffic_simulator/entity/entity_base.hpp

```diff
@@ -93,9 +93,9 @@ class EntityBase
 
   virtual auto getGoalPoses() -> std::vector<traffic_simulator_msgs::msg::LaneletPose> = 0;
 
-  virtual auto getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel;
+  virtual auto getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel = 0;
```

Comment:
```
virtual auto getDriverModel() -> const traffic_simulator_msgs::msg::DriverModel = 0
```
↓
```
virtual auto getDriverModel() const -> traffic_simulator_msgs::msg::DriverModel = 0
```

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/650/commits/d3a8c09835cca7d0b623b86317f54f9152eb257e